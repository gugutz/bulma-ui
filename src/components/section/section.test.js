import React from 'react'
import renderer from 'react-test-renderer'
import Section from './section'

describe('<Section /> component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Section>text</Section>).toJSON()
    expect(tree).toMatchSnapshot()
  })

  const sizes = ['normal', 'medium', 'large']

  for (const size of sizes) {
    it('should support size prop with ' + size, () => {
      const tree = renderer.create(<Section size={size}>text</Section>).toJSON()
      expect(tree).toMatchSnapshot()
    })
  }
})
