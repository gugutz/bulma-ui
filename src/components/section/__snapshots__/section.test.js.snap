// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Section /> component should render correctly 1`] = `
<div
  className="section"
>
  text
</div>
`;

exports[`<Section /> component should support size prop with large 1`] = `
<div
  className="section is-large"
>
  text
</div>
`;

exports[`<Section /> component should support size prop with medium 1`] = `
<div
  className="section is-medium"
>
  text
</div>
`;

exports[`<Section /> component should support size prop with normal 1`] = `
<div
  className="section"
>
  text
</div>
`;
